x-pack/plugins/uptime/public/components/settings/add_connector_flyout.tsx

```diff
@@ -48,6 +33,11 @@ export const AddConnectorFlyout = ({ focusInput }: Props) => {
     focusInput();
   }, [addFlyoutVisible, dispatch, focusInput]);
 
+  const ConnectorAddFlyout = triggersActionsUi.getAddConnectorFlyout({
+    consumer: 'case',
```

Comment:
Should this be `uptime`?

```suggestion
    consumer: 'uptime',
```